### DISCUSSION TITLE
Social Icons no longer present in footer of my personal site

### DISCUSSION THREAD

**Discussion by jwdinius** [alshedivat/al-folio#1987] [Q&A] [answered]
Thanks for a great Jekyll theme!  I recently updated my personal site to use the latest version of al-folio and, after the integration, I notice that my social icons are no longer present at the bottom of my `about` page despite `social: true` being set in the page's header as well as the usernames being populated the same as they were.  Does anyone have any advice for where I should look to troubleshoot this issue?  Thanks!

For those that are curious:
[Personal site](https://jwdinius.github.io) (see that no social icons are present at the bottom of the page)
[GitHub repo](https://github.com/jwdinius/jwdinius.github.io) (check the `_config.yml` and see that social fields are filled in, and `social: true` is set in `about.md`)

**Comment by george-gca** [comment]
One thing that is missing is the updated icons names. In #1955 I updated fontawesome, so the icons names must be changed. For example, `fab fa-github` is called `fa-brands fa-github` in the new version. Make sure to get the latest update to see if it works.

**Reply by jwdinius** [reply]
Thanks for your response @george-gca.  I [cherry-picked](https://github.com/jwdinius/jwdinius.github.io/commit/ab7a4d09ff9f6eb75803795181c2b53240c336eb) your changes from the master branch after they were merged but I'm still not seeing the social icons.

**Reply by jwdinius** [reply, marked as answer]
I did get it working now after [merging al-folio/master into mine](https://github.com/jwdinius/jwdinius.github.io/commit/ea1532c836c3f61d650c73c97d7c853858fdf172). I suspect that a style definition file (under `assets`) got messed up when I did a manual upgrade earlier this year.
